src/uiConfig/index.tsx

```diff
@@ -7,10 +8,20 @@ import { AddEditEntity } from "../pages/components/AddEditEntity";
 import { ListEntities } from "../pages/components/ListEntities";
 import { ShowEntity } from "../pages/components/ShowEntity";
 import SearchList from "../pages/components/SearchList";
-// import {MainApp} from '../pages/screens/MainApp'
+import { OrderLineView } from "../pages/components/OrderLineView";
+
+// TSD WEB APP INTEGRATION
+import { HeaderComponent } from "../pages/components/HeaderComponent";
+import { OrderSearchList } from "../pages/components/OrderSearchList";
```

Comment:
Order Search related parts , remove them

Reply:
Removed the tsd part component